bool is_following
---
bool is_successful